ert_storage/endpoints/compute/misfits.py

```diff
@@ -33,20 +34,25 @@ async def get_response_misfits(
     Compute univariate misfits for response(s)
     """
 
-    ensemble = db.query(ds.Ensemble).filter_by(id=ensemble_id).one()
-    reponse_query = (
+    response_query = (
         db.query(ds.Record)
+        .filter(ds.Record.observations != None)
```

Comment:
you claimed to be consistent wrt `is not None` vs `!= None` . Is there a reason this code snippet should be written like this ?

Reply:
1. It's the "original" code, so I guess it's fine if it stays
2. Not sure if `filter` method accepts `is not None` construct - will try

Reply:
Because `ds.Record.observations` isn't `None`. SQLAlchemy overloads the comparison to `None` and interprets it as the SQL query where `observations` is not `NULL`.

`ds.Record.observations is not None` would get evaluated to `False` and `.filter(False)` is nonsensical.

Reply:
1. yes - i see now... my bad! 🙈